oniux-git/lilac.yaml

```
@@ -0,0 +1,17 @@
+maintainers:
+  - github: Giteeajake
+    email: giteeajake@archlinuxcn.org
+  - github: Integral-Tech
+    email: integral@archlinuxcn.org
+
+pre_build_script: |
+  update_pkgrel()
+  vcs_update()
+
+post_build_script: |
+  git_pkgbuild_commit()
+  update_aur_repo()
+
+update_on:
+  - source: gitlab
+    host: gitlab.torproject.org/tpo/core/oniux
```

```suggestion
  - source: gitlab
    host: gitlab.torproject.org
    gitlab: tpo/core/oniux
```

https://nvchecker.readthedocs.io/en/latest/usage.html#check-gitlab